gulpfile.babel.js

```diff
@@ -46,6 +49,16 @@ function bundle() {
         .pipe(gulp.dest('./'));
 }
 
+function compress(cb) {
+  pump([
+        gulp.src('compiled.js'),
+        uglify(),
+        gulp.dest('./')
```

Comment:
Overwriting compiled.js is fine.  I think sourcemaps still work with minified JS and even if they dont we'll notice when we debug and then we can just generate it again without minifying.